posthog/constants.py

```diff
@@ -303,9 +303,11 @@ class FlagRequestType(StrEnum):
 
 ENRICHED_DASHBOARD_INSIGHT_IDENTIFIER = "Feature Viewed"
 DATA_WAREHOUSE_TASK_QUEUE = "data-warehouse-task-queue"
-BATCH_EXPORTS_TASK_QUEUE = "no-sandbox-python-django"
+BATCH_EXPORTS_TASK_QUEUE = "batch-exports-task-queue"
+SYNC_BATCH_EXPORTS_TASK_QUEUE = "no-sandbox-python-django"
```

Comment:
Bonus: When we finish migrating everything to async we can just get rid of the "no-sandbox-python-django" queue.